next.config.js

```diff
@@ -15,15 +15,32 @@ const securityHeaders = [
   },
 
   // adds x-frame-options
-  {
-    key: 'X-Frame-Options',
-    value: 'SAMEORIGIN'
-  },
+  // remove for safe app to load
+  // {
+  //   key: 'X-Frame-Options',
+  //   value: 'SAMEORIGIN'
```

Comment:
Should we restore this?

Reply:
We won't be able to load the app in the safe app iframe with it set. 

> if you specify SAMEORIGIN, you can still use the page in a frame as long as the site including it in a frame is the same as the one serving the page.

I'm not sure when or why it got added. I guess click-jacking is perhaps the security concern.

https://www.keycdn.com/blog/x-frame-options